modules/client-configuration/pages/vhm.adoc

```diff
@@ -32,4 +32,9 @@ If all of your VMs are provided by the same zone, you are within the terms and c
 If your VMs are provided by different zones, even if they are within the same region, you might not meet the conditions of the ``1-2 Virtual Machines`` subscription.
 In this case, check your subscription carefully.
 
+[NOTE]
+====
+Only BYOS (Bring-Your-Own-Subscription) instances are passed to the subscription matcher. In case your Public Cloud instances are PAYG (Pay-As-You-Go), these will be excluded from the subscription matcher counting.
```

Comment:
```suggestion
Only instances using bring your own subscription (BYOS) are passed to the subscription matcher. 
If your public cloud instances are pay as you go (PAYG), they will be excluded from the subscription matcher counting.
```

Reply:
I'm not that happy with your suggestion ;-)  I assume that these instances are prominently labeled either BYOS or PAYG, thus it might make sense to have this in the running text. This is a tough reading for the international readers, IMO:
> If your public cloud instances are pay as you go ...
I'll provide a suggestion, too ;)

Reply:
Sure :) I was mostly unhappy about the hyphens and initcaps, and then decided to turn it around.